packages/compass-components/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
I'd start right away to put these in `ROOT/config/` as we do in mongosh and require these here already, so with the next package can just copy paste it.

Reply:
FWIW I think it would be better to move them into a workspace inside the monorepo (like `packages/compass-config` or something), that way all config specific dependencies can stay in their own `package.json` instead of bundling up in the root one (same thing I think we should do for the root-level scripts)

Reply:
But also personally I think not moving stuff right away to other places in this PR makes it easier to review and we can do a follow up that makes it available for other workspaces

Reply:
As long as we don't start other packages with different configurations that would be ok

Reply:
Holding off for now, let's be sure to move it into a centralized place then when we're adding another package.